modules/openapi-generator/src/main/resources/r/api.mustache

```diff
@@ -153,7 +153,11 @@
 
       {{/vendorExtensions.x-streaming}}
       if (local_var_response$status_code >= 200 && local_var_response$status_code <= 299) {
-        local_var_response$content
+        if (is.raw(local_var_response$content)) {
+          return(local_var_response)
+        } else {
+          return(local_var_response$content)
+        }
```

Comment:
@wing328 feel free to keep or discard this change. As requested I pulled this out of #20131. If declined, the result will be that when an endpoint that returns binary data (e.g., a gzipped directory) that cannot be coerced to text, the return value will be a raw vector. If accepted, the return value in this case will be an instance of `ApiResponse`, which is what gets returned if there are error codes too.

Reply:
Another alternative would be to `return(invisible(local_var_response$content))` so that the raw vector, though returned, will not be printed in the event that it is not captured to a variable. This may be a better option since it seems likely that these sorts of endpoints will be invoked without assigning the response to a variable.